package.json

```diff
@@ -15,10 +16,13 @@
     "debug": "^4.1.1",
     "koa": "^2.7.0",
     "koa-session": "^5.12.2",
+    "koa-static-server": "^1.4.0",
+    "mashlib": "^1.0.0",
     "redis": "^2.8.0",
-    "solid-idp": "^0.1.2",
-    "wac-ldp": "^0.6.0",
-    "websockets-pubsub": "^0.3.2",
+    "solid-idp": "github:inrupt/solid-idp#dev",
+    "solid-server-ts": "github:inrupt/solid-server-ts#dev",
+    "wac-ldp": "github:inrupt/wac-ldp#dev",
```

Comment:
Should we be pushing to master when these are pointing to dev branches?

Reply:
fixed, we're only keeping the solid-server-ts as dev from git.